bundles/bundle.properties

```diff
@@ -91,158 +91,158 @@ sector.exoprosopa-19s-01-newgrounds.name = Newgrounds
 sector.exoprosopa-19s-01-newgrounds.description = Your spaceship sent you to this sector as your first mission.\nThis sector is protected by a small base.\nAnalyze this sector and build infrastructure to protect yourself.\n\n[accent]Can you.. Changes the fate ?
 
 sector.exoprosopa-19s-02-mountain-ridge.name = Mountain Ridge
-sector.exoprosopa-19s-02-mountain-ridge.description = This sector is the only path available from [accent]Newgrounds[].\nThe enemy will send ground forces to take down your base.\n\nThis sector is rich in [accent]tantrium[].\nResearch [accent]Elixir Purifier[] and [accent]Tantrium Crusher[] to survive from the enemies.
+sector.exoprosopa-19s-02-mountain-ridge.description = This sector is the only path available from [accent]Newgrounds[].\nThe enemy will send ground forces to take down your base.\n\nThis sector is rich in [accent]Tantrium[].\nResearch [accent]Elixir Purifier[] and [accent]Tantrium Crusher[] to survive from the enemies.
 
 sector.exoprosopa-19s-03-a-luminary.name = Luminary
-sector.exoprosopa-19s-03-a-luminary.description = Due to lack of vents, you will need a better source of power.\nResearch [accent]Burner Generator[] to increase your power production.\n\nIt might be a good idea to have air defences protecting bases weak points.
+sector.exoprosopa-19s-03-a-luminary.description = Due to lack of vents, you will need a better source of power.\nResearch the [accent]Burner Generator[] to increase your power production.\n\nIt might be a good idea to have air defence protecting bases weak points.
```

Comment:
so, we must add a single air defense/defence as brits to protect the core?, it kinda bri'ish.